lib/ansible/modules/cloud/azure/azure_rm_storageaccount.py

```diff
@@ -59,15 +58,16 @@
             - type
     custom_domain:
         description:
-            - User domain assigned to the storage account. Must be a dictionary with 'name' and 'use_sub_domain'
-              keys where 'name' is the CNAME source. Only one custom domain is supported per storage account at this
-              time. To clear the existing custom domain, use an empty string for the custom domain name property.
+            - User domain assigned to the storage account.
+            - Must be a dictionary with I(name) and I(use_sub_domain) keys where I(name) is the CNAME source.
+            - Only one custom domain is supported per storage account at this time.
+            - To clear the existing custom domain, use an empty string for the custom domain name property.
             - Can be added to an existing storage account. Will be ignored during storage account creation.
         aliases:
             - custom_dns_domain_suffix
     kind:
         description:
-            - The 'kind' of storage.
+            - The I(kind) of storage.
```

Comment:
Since this is the current option being described, I think I would leave the `I(..)` out here.

```suggestion
            - The kind of storage.
```